src/store/index.js

```diff
@@ -23,17 +23,65 @@ import Vue from 'vue'
 import Vuex from 'vuex'
 
 import {
+	PRIORITY_INBOX_ID,
 	UNIFIED_ACCOUNT_ID,
 	UNIFIED_INBOX_ID,
-	PRIORITY_INBOX_ID,
 } from './constants'
 import actions from './actions'
 import { getters } from './getters'
 import mutations from './mutations'
-
+import { normalizedEnvelopeListId } from './normalization'
+import orderBy from 'lodash/fp/orderBy'
 Vue.use(Vuex)
 
+const moduleThreads = {
+	state: {
+		lists: {
+			[UNIFIED_INBOX_ID]: {},
+			[PRIORITY_INBOX_ID]: {},
+		},
+	},
+	mutations: {
+		addAccount(state, account) {
+			const mailboxes = account.mailboxes || []
+			mailboxes.map(mailboxId => Vue.set(state.lists, mailboxId, {}))
+		},
+		addMailbox(state, { mailbox }) {
+			Vue.set(state.lists, mailbox.databaseId, {})
+		},
+		addEnvelope(state, { query, envelope }) {
```

Comment:
When a vuex module is not namespaced it reacts to the existing mutations. A commit for addEnvelope trigger the existing mutation and our new one.